### ISSUE TITLE
RFE: Search

### ISSUE THREAD

**Issue by techtonik** [difio/difio#8]
There is no search / list of packages tracked in last 6 months.

**Comment by atodorov** [comment]
Hi Anatoly,
can you be more specific. Packages older than 6 months are shown with orange or red color and a tooltip is also available. 

What exactly are you refering to wrt this issue?

**Comment by techtonik** [comment]
I mean that information on the main page is limited to 7 names only, with
no way to see "22004 more analyzed" packages or search for them.

**Comment by atodorov** [comment]
This is correct, there's no search right now. In fact you are the first one to request it :)

As a work around you may import some package names/versions and see what's available at Difio.

**Comment by techtonik** [comment]
> This is correct, there's no search right now. In fact you are the first
> one to request it :)
> 
> It is because nobody knows what dif.io is. =) I could miss it too, but I
> was curious to
> click on my astdump package name.

So a box where people can check their package without registration would be
rather nice.

> As a work around you may import some package names/versions and see what's
> available at Difio.
> 
> The issue #7 is just about that. =) I mean - I can't. I entered some names,
> yes, but only the package with actual update has some information. Other
> names - the one that is correct and update, and the one that is incorrect
> are both the same.

**Comment by atodorov** [comment]
> So a box where people can check their package without registration would be rather nice.

Thanks for the feedback, this can easily be implemented, I will take a look at it.

**Comment by atodorov** [comment]
> The issue #7 is just about that. =) I mean - I can't. I entered some names, yes, but only the package with actual > update has some information. Other names - the one that is correct and update, and the one that is incorrect are > both the same.

I think I got you now, you mean if the package is up to date, to have a link to the previous analytics to see what it was like before the latest version. Is this correct? 

And in the case you have entered an invalid package name or non existing version there's nothing we can do. There's no way to know if the package is up-to-date or not existing or nobody used it before that (e.g. brand new package).

**Comment by techtonik** [comment]
> I think I got you now, you mean if the package is up to date, to have a
> link to the previous analytics to see what it was like before the latest
> version. Is this correct?
> 
>  Exactly.
> 
>  And in the case you have entered an invalid package name or non existing
> version there's nothing we can do. There's no way to know if the package is
> up-to-date or not existing or nobody used it before that (e.g. brand new
> package).
> 
> Right. But something still can be done for a user - show package processing
> status in a style/link of its name - like this:
1. up-to-date   -- no specific color/style, perhaps a "up to date" tooltip,
   link to previous analytics
2. in-progress  -- data is not yet ready
3. update-available     -- color/style mark, link leads to comparison with
   latest version
4. unknown     -- color/style mark that package is not found in known
   sources
